tests/yspryn/test_fraction_hw9.py

```diff
@@ -1,4 +1,5 @@
 from yspryn.hw9.fraction import Fraction
+import pytest
```

Comment:
as per codestyle:

first imports from built in modules, 
then import from installed modules, e.g. pytest, requests, selenium
then only imports from project modules